### DISCUSSION TITLE
When will Docker Hub synchronize and update version 5.02

### DISCUSSION THREAD

**Discussion by yisirman** [SoftEtherVPN/SoftEtherVPN#2027] [Q&A] [answered]
https://hub.docker.com/r/softethervpn/vpnserver
May I ask when this warehouse can be updated to version 5.02 ?  thanks.

**Comment by chipitsine** [comment]
@weidi , what is the current status ? are we good now ?

**Reply by weidi** [reply, marked as answer]
From my point of view yes but stable branch is an issue

**Comment by shakibamoshiri** [comment]
@chipitsine 
I am using SE dev on docker compose and kubernetes more than a year 
it is reliable 
can I make a pull request to add docs, etc to support k8s build and configurations?

**Reply by chipitsine** [reply]
sure
